tests/steps/test_cli.py

```diff
@@ -0,0 +1,50 @@
+# coding=utf-8
+"""CLI for report builder feature tests."""
+import pathlib
+import shutil
+
+from bs4 import BeautifulSoup
```

Comment:
In this case, bs4 shall be defined as a test dependency. See the distinction of requirements implemented in the rdf-differ project.